Controller/AuthorizeController.php

```diff
@@ -127,7 +127,7 @@ public function __construct(
      */
     public function authorizeAction(Request $request): Response
     {
-        $user = $this->tokenStorage->getToken()->getUser();
+        $user = $this->tokenStorage->getToken()?->getUser();
```

Comment:
Perhaps old-fashioned null-check, since it's a  min. php 7.4+ ?
```php
$user = null;
if( null !== $token = $this->tokenStorage->getToken())
{
   $user = $token->getUser();
}
```

Reply:
good point